repository/src/main/java/org/alfresco/service/cmr/repository/ContentService.java

```diff
@@ -210,10 +211,48 @@ default DirectAccessUrl requestContentDirectUrl(NodeRef nodeRef, boolean attachm
      * @param propertyQName the name of the property, which must be of type <b>content</b>
      * @return Returns a key-value (String-String) collection of storage headers/properties with their respective values for a given {@link NodeRef}.
      */
-    @Auditable
+    @Auditable(parameters = {"nodeRef", "propertyQName"})
     @Experimental
     default Map<String, String> getStorageProperties(NodeRef nodeRef, QName propertyQName)
     {
         return Collections.emptyMap();
     }
+
+    /**
+     * Submit a request to send content to archive (offline) state.
+     * If no connector is present or connector is not supporting sending to archive, then {@code false} will be returned.
+     * Specific connector will decide which storage class/tier will be set for content.
+     * This method is experimental and subject to changes.
+     *
+     * @param nodeRef a reference to a node having a content property
+     * @param propertyQName the name of the property, which must be of type <b>content</b>
+     * @return true when request successful, false when unsuccessful.
+     */
+    @Auditable(parameters = {"nodeRef", "propertyQName"})
+    @Experimental
+    default boolean requestSendContentToArchive(NodeRef nodeRef, QName propertyQName)
```

Comment:
I can see that we have not provided defaults for methods such as `requestContentDirectUrl(NodeRef nodeRef, boolean attachment, Long validFor);` Should we be following this pattern and force implementing class to override your new methods too? (I guess this also includes `getStorageProperties(NodeRef nodeRef, QName propertyQName)`)
Happy either way, I just like to rock the boat :smile:

Reply:
As discussed - let's stay with default methods.